service/payment.go

```diff
@@ -32,6 +32,9 @@ type PaymentService struct {
 // PaymentStatus Enum Type
 type PaymentStatus int
 
+// PaymentSessionKind
+const PaymentSessionKind = "payment-session#payment-session"
```

Comment:
As agreed, this needs to be looked at as a separate ticket as the spec needs to be updated with a real enum type.  We can however check the mechanism works and returns this hardcoded type for the purpose of checking the code matches the spec

Reply:
:+1: